### DISCUSSION TITLE
[Question] How Production ready is Flet?

### DISCUSSION THREAD

**Discussion by Tejaswgupta** [flet-dev/flet#2199] [Q&A]
I've been working with Flutter for a long time now and the time for its maturity for also considerable. Now primarily working with AI application Python is the go-to, I've tried researching into ffi and interops(since we need offline only functionality) with Python, rust and C++ . Looking at Flet , it looks pretty promising , how production ready apps can be built with it? Say for ex a Speech-to-text app with Whisper.cpp without significant performance drop and crashes.

**Comment by ndonkoHenri** [comment]
Flet is atm not yet very [stable](https://flet.dev/roadmap/). (It's a work in progress)
Many cool apps have been done with it though.

Check this out (it was made with whisper): https://github.com/flet-dev/flet/discussions/659

**Reply by baseplate-admin** [reply]
Hey one side, question. When can we expect standalone android apps?

**Reply by FeodorFitsner** [reply]
By the end of this month.
